README.markdown

```diff
@@ -45,7 +45,7 @@ conduct.
 Please adhere to this code of conduct in any interactions you have in the
 Jekyll community. It is strictly enforced on all official Jekyll
 repositories, websites, and resources. If you encounter someone violating
-these terms, please let a [team captain](https://github.com/orgs/jekyll/teams/affinity-team-captains/members) know and we will address it as soon as possible.
+these terms, please let one of our core team members [Olivia](mailto:olivia@jekyllrb.com?subject=Jekyll%20CoC%20Violation), [Pat](mailto:pat@jekyllrb.com?subject=Jekyll%20CoC%20Violation), [Matt](mailto:matt@jekyllrb.com?subject=Jekyll%20CoC%20Violation) or [Parker](mailto:parker@jekyllrb.com?subject=Jekyll%20CoC%20Violation) know and we will address it as soon as possible.
```

Comment:
I'm not sure that `jekyllrb.com` email addresses are set up or are they forwarded to our other emails?

Reply:
@mattr-  @parkr kindly offered to configure this. 📫

Reply:
oh, excellent. i missed that comment in the other issue.

Reply:
All set up! You too, @DirtyF 😄 I went with first names – if you'd prefer something else, please let me know!

Reply:
@parkr 🍷 cheers mate!